### PR TITLE
Include 'try it on katacoda' on mainpage

### PR DESCRIPTION
**Pull request by iranzo**
**What this PR does / why we need it**:

Enables 'try it' on katacoda so that users can test it on an online system for free

**Does this PR fix any issue?** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
> Fixes #

**Special notes for your reviewer**:
image

### CODE REVIEW

**Comment by codificat** [comment]
/hold
until this is available in a public URL that isn't a personal account - and hopefully without requiring registration

File: _layouts/homepage.html [hunk 1]
```diff
@@ -17,6 +17,19 @@ <h1 class="hero-overlay-text">Building a virtualization API for Kubernetes</h1>
         </div>
         <div class="row align-items-center justify-content-center">
           <div class="card-group">
+                        <div class="card" style="margin-right: .2rem;">
```
Comment (tripledes) [inline _layouts/homepage.html]: Could you please fix the indentation for this new block to be aligned with the other blocks? Thanks.

Comment (iranzo) [reply _layouts/homepage.html]: done, thx

**Comment by iranzo** [comment]
Url is now the organization account, so this should not longer be an issue @codificat

**Comment by codificat** [comment]
We're still waiting for a proper location for the scenario itself (its source code).

Once it's there let's have a review of the scenario before it gets public as the most visible component of the home page.

**Comment by iranzo** [comment]
https://github.com/kubevirt/katacoda-scenarios/pull/2 will merge the contents for the scenario

**Pull request merged**